export_img.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import subprocess
+from pathlib import Path
+
+base = Path(".")
```

Comment:
I u base, i u inkscape by se hodil komentar vysvetlujici, co se zde ma/muze doplnit. 
Napr. "base: nech "." pokud chces spustit program v adresari, kde mas svg soubory, nebo dopln vlastni cestu k obrazkum"

Reply:
Díky, nikdy jsem program takto nepsala tak mi to nedošlo a zapomněla jsem, doplním.

Reply:
doplněno